include/seqan3/alphabet/structure/structured_rna.hpp

```diff
@@ -128,7 +128,7 @@ class structured_rna :
     //!\name Write functions
     //!\{
     //!\brief Assign from a nucleotide character. This modifies the internal sequence letter.
-    constexpr structured_rna & assign_char(char_type const c)
+    constexpr structured_rna & assign_char(char_type const c) noexcept
```

Comment:
Should this be part of the PR as it does not consider the constructor? I checked that e.g. masked and quality do not have  the noexpect specifier at their assign_char function either..

Reply:
fixed masked but didnt find an issue with quality...?

Reply:
sorry, it was qualified but you already fixed that.